### PR TITLE
Logitravel dynamic packages

### PR DESCRIPTION
**Pull request by michalsanger**
I know no tests :-( Will add them ASAP, just need this feature on production to unblock FE guys, so please have mercy :-)

### CODE REVIEW

File: src/dynamicPackage/dataloaders/DynamicPackages.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        Itinerary: 1,
+        TransportNumbers: params.inboundFlights,
+      },
+    ],
+    Travellers: {
+      Seniors: 0,
+      Adults: params.passengers.adults || 0,
+      Children: params.passengers.infants || 0,
+    },
+    Language: 'en', // FIX: use language input
```
Comment (mrtnzlml) [inline src/dynamicPackage/dataloaders/DynamicPackages.js]: @michalsanger Did you leave it here intentionally?

Comment (michalsanger) [reply src/dynamicPackage/dataloaders/DynamicPackages.js]: Yes, so far OK with `en` only

File: src/dynamicPackage/queries/AllDynamicPackages.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import { GraphQLNonNull } from 'graphql';
+import { connectionArgs, connectionDefinitions } from 'graphql-relay';
+import { connectionFromArray } from '../../common/services/ArrayConnection';
+import Currency from '../../common/types/enums/Currency';
+import { post } from '../../common/services/HttpRequest'; //eslint-disable-line no-unused-vars
```
Comment (mrtnzlml) [inline src/dynamicPackage/queries/AllDynamicPackages.js]: Why is it needed?

Comment (michalsanger) [reply src/dynamicPackage/queries/AllDynamicPackages.js]: no idea. removed

**Pull request merged**